src/utils.ts

```
@@ -156,33 +142,29 @@ export type LocaleConfig<T = string[] | LocaleObject[]> = { langDir: string; loc
-      delete locale.file
-
-      // merge locale and files with existing entry
-      const merged = mergedLocales.get(locale.code)
-      if (merged != null) {
-        merged.files ??= [] as LocaleFile[]
-        // @ts-ignore
-        merged.files.unshift(...files)
-        mergedLocales.set(locale.code, assign({}, locale, merged))
-        continue
+      const files = isString(locale) ? [] : resolveRelativeLocales(current, config)
+      delete current.file
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)